boggle_solver.js

```diff
@@ -121,22 +123,18 @@ var wordSearch = function(row, col, grid, visited, trieTree, foundWord, solution
 		[1, -1],
 		[-1, 1]];
   
-	// console.log(row, col, grid[row][col], visited);
 	if (row < 0 || col < 0 ||
       row >= grid.length || col >= grid.length ||
-      visited[row][col]) {
+      visited[row][col] == true) {
```

Comment:
if (row < 0 || col < 0 || row >= grid.length || col >= grid.length || visited[row][col] == true) could be its own boolean variable (e.g. isInvalidInput) for readability